tests/smt2/qf_ufbv/test-define-sort.smt2

```diff
@@ -0,0 +1,8 @@
+(set-logic QF_UFBV)
+(define-sort Int () (_ BitVec 32))
```

Comment:
I recommend:
(a) having another less confusing example (e.g., calling it `BV`).
(b) keep this example, but also include a comment that says it tests a special case that is allowed by smtlib.

Reply:
Good idea! Done in 03331c979f6c972fe0924e6c471856f332ce1cab.